reference/6/Microsoft.PowerShell.Core/About/about_Thread_Jobs.md

```diff
@@ -18,36 +18,59 @@ the current session process.
+jobs types provided by PowerShell to support concurrency.
 
-Start a thread job by using the `Start-ThreadJob` cmdlet. This cmdlet is
-available in the **ThreadJob** module that ships with PowerShell.
-`Start-ThreadJob` returns a single job object that encapsulates the running
-command or script, and can be used with all PowerShell job manipulating
-cmdlets.
+- `RemoteJob` - Commands and scripts run in a remote session. For information,
+  see [about_Remote_Jobs](about_Remote_Jobs.md).
+- `BackgroundJob` - Commands and scripts run in a separate process on the local
+  machine. For more information, see [about_Jobs](about_Jobs.md).
+- `PSTaskJob` or `ThreadJob` - Commands and script runs in a separate thread
+  within the same process on the local machine.
 
-## The job cmdlets
+Thread based jobs are not as robust as Remote and Background jobs, because they
+run in the same process on different threads. If one job has a critical error
+that crashes the process, then all other jobs in the process are terminated.
+
+However, thread-based jobs require less overhead. They don't use the remoting
+layer or serialization. The result objects are returned as references to live
+objects in the current session. Without this overhead, thread-based jobs run
+faster and use fewer resources than the other job types.
+
+> [!IMPORTANT]
+> The parent session that created the job monitors the job status and collects
+> pipeline data. The job child process is terminated by the parent process once
+> the job reaches a finished state. If the parent session is terminated, all
+> running child jobs are terminated along with their child processes.
+
+There are two ways work around this limitation:
+
+1. Use `Invoke-Command` to create jobs that run in disconnected sessions. For
+   more information, see [about_Remote_Jobs](about_Remote_Jobs.md).
+1. Use `Start-Process` to create a new process rather than a job. For more
+   information, see
+   [Start-Process](xref:Microsoft.PowerShell.Management.Start-Process).
+
+## How to start and manage thread-based jobs
 
-|Cmdlet           |Description                                            |
-|-----------------|-------------------------------------------------------|
-|`Start-ThreadJob`|Starts a thread job on a local computer.               |
-|`Get-Job`        |Gets the jobs that were started in the current session.|
-|`Receive-Job`    |Gets the results of jobs.                              |
-|`Stop-Job`       |Stops a running job.                                   |
-|`Wait-Job`       |Suppresses the command prompt until one or all jobs are|
-|                 |complete.                                              |
-|`Remove-Job`     |Deletes a job.                                         |
+There are two ways to start thread-based jobs:
 
-## How to start a thread job on the local computer
+- `Start-ThreadJob` - from the ThreadJob module
+- `ForEach-Object -Parallel -AsJob` - the parallel features added in PowerShell
```

Comment:
Should feature be plural? I'd say it was a single feature, but it was added as a number of parameters on cmdlets.

Reply:
Fixed